packages/cli/src/commands/alias/ls.ts

```diff
@@ -73,21 +72,20 @@ export default async function ls(client, opts, args) {
     );
   }
 
-  now.close();
   return 0;
 }
 
-function printAliasTable(aliases) {
+function printAliasTable(aliases: Alias[]) {
   return `${table(
     [
-      ['source', 'url', 'age'].map(h => chalk.gray(h)),
+      ['source', 'url', 'age'].map(header => chalk.gray(header)),
       ...aliases.map(a => [
         // for legacy reasons, we might have situations
         // where the deployment was deleted and the alias
         // not collected appropriately, and we need to handle it
         a.deployment && a.deployment.url ? a.deployment.url : chalk.gray('–'),
         a.alias,
-        ms(Date.now() - new Date(a.createdAt)),
+        ms(Date.now() - new Date(a.created).getTime()),
```

Comment:
This should remain `createdAt`

Reply:
The problem is `createdAt` does not exist in the `Alias` type. TS complained earlier because createdAt doesn't exist. What we can do are 2 things.

1.- Create `createdAt` property in the `Alias` type
2.- We could rename `created` to` createdAt `function scope

Reply:
What do you think @styfle  ? 🤔

Reply:
It sounds like the type is wrong. It should have `createdAt` and thats what the code was using before